runtime/src/accounts_db.rs

```diff
@@ -1791,7 +1791,7 @@ struct ShrinkStats {
     store_accounts_elapsed: AtomicU64,
     update_index_elapsed: AtomicU64,
     handle_reclaims_elapsed: AtomicU64,
-    write_storage_elapsed: AtomicU64,
+    remove_old_stores_shrink: AtomicU64,
```

Comment:
new name doesn't imply this is a time, add `_elapsed` or maybe better yet `_us`?